pkg/virt-handler/vm.go

```diff
@@ -344,10 +344,20 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 				domainInterfaceStatusByMac[domainInterfaceStatus.Mac] = domainInterfaceStatus
 			}
 
+			existingInterfacesSpecByName := map[string]v1.Interface{}
+			for _, existingInterfaceSpec := range vmi.Spec.Domain.Devices.Interfaces {
+				existingInterfacesSpecByName[existingInterfaceSpec.Name] = existingInterfaceSpec
+			}
+
 			// Iterate through all domain.Spec interfaces
 			for _, domainInterface := range domain.Spec.Devices.Interfaces {
 				interfaceMAC := domainInterface.MAC.MAC
 				var newInterface v1.VirtualMachineInstanceNetworkInterface
+				var isForwardingBindingInterface bool = false
+
+				if existingInterfacesSpecByName[domainInterface.Alias.Name].Masquerade != nil {
```

Comment:
My comment about slirp handling is still pending

Reply:
Missed that, added now